packages/mom5/package.py

```diff
@@ -533,53 +592,50 @@ def edit(self, spec, prefix):
 	hpm -r -o $*.hpm $*.x
 """
 
-        fullconfig = config[self.compiler.name] + config["post"]
+        fullconfig = config[pkg.compiler.name] + config["post"]
         print(fullconfig)
         with open(makeinc_path, "w") as makeinc:
             makeinc.write(fullconfig)
 
-    def build(self, spec, prefix):
+    def build(self, pkg, spec, prefix):
 
         # cd ${ACCESS_OM_DIR}/src/mom/exp
         # export mom_type=ACCESS-OM
         # ./MOM_compile.csh --type $mom_type --platform spack
-        with working_dir(join_path(self.stage.source_path, "exp")):
+        with working_dir(join_path(pkg.stage.source_path, "exp")):
             build = Executable("./MOM_compile.csh")
 
-            if self.spec.satisfies("+restart_repro"):
+            if pkg.spec.satisfies("+restart_repro"):
                 build.add_default_env("REPRO", "true")
                 print("INFO: +restart_repro applied")
 
-            if self.__builds[self.__version]["gtracers"]:
-                build.add_default_env("SPACK_GTRACERS_EXTERNAL", "true")
-
             if self.__version != "access-esm1.5":
                 # The MOM5 commit d7ba13a3f364ce130b6ad0ba813f01832cada7a2
                 # requires the --no_version switch to avoid git hashes being
                 # embedded in the binary.
                 build.add_default_arg("--no_version")
-                if self.spec.satisfies("+optimisation_report"):
+                if pkg.spec.satisfies("+optimisation_report"):
                     build.add_default_env("REPORT", "true")
                     print("INFO: +optimisation_report applied")
 
             build(
                 "--type",
-                self.__builds[self.__version]["type"],
+                self.__builds[self.__version],
                 "--platform",
                 self.__platform,
                 "--no_environ"
             )
 
-    def install(self, spec, prefix):
+    def install(self, pkg, spec, prefix):
 
         mkdirp(prefix.bin)
         install(
             join_path(
                 "exec",
                 self.__platform,
-                self.__builds[self.__version]["type"],
-                "fms_" + self.__builds[self.__version]["type"] + ".x"
+                self.__builds[self.__version],
+                "fms_" + self.__builds[self.__version] + ".x"
             ),
             prefix.bin
         )
-        install(join_path("bin", "mppnccombine." + self.__platform), prefix.bin)
+        install(join_path("bin", "mppnccombine." + self.__platform), prefix.bin)
```

Comment:
Your editor has removed a newline. Can you please add it back?

Reply:
I think this file originally didn't have a new line, and Github was saying it should. Regardless, fixed in [45a1981](https://github.com/ACCESS-NRI/spack-packages/pull/238/commits/45a19817aee613637f25091560d4fa76b49a8623)

Reply:
The line above originally had a newline after the `)`, but it appears to have been removed by your editor. That's why Github displays that the line has changed even though there are no changes to the alphanumeric characters.